script/common.sh

```diff
@@ -23,16 +23,11 @@ function check_changed_packages() {
     return 1
   fi
 
-  # Filter out any packages that don't have a pubspec.yaml: they have probably
-  # been deleted in this PR. Also filter out `location_background` since it
-  # should be removed soon.
   CHANGED_PACKAGES=""
   CHANGED_PACKAGE_LIST=()
   for package in "${packages[@]}"; do
-    if [ -f "$REPO_DIR/packages/$package/pubspec.yaml" ] && [ $package != "location_background" ]; then
-      CHANGED_PACKAGES="${CHANGED_PACKAGES},$package"
-      CHANGED_PACKAGE_LIST=("${CHANGED_PACKAGE_LIST[@]}" "$package")
-    fi
+    CHANGED_PACKAGES="${CHANGED_PACKAGES},$package"
```

Comment:
Seems like the pubspec.yaml logic was introduced to prevent a deleted package to be running on CI, maybe we can just check that the directory exists instead?

cc @gspencergoog who originally introduce this logic.

Reply:
Changed this to check if the directory exists